src/components/SignerPanel/SignerPanel.js

```diff
@@ -360,12 +352,10 @@ class SignerPanel extends React.PureComponent {
                     >
                       <Screen>
                         <ValidateWalletWeb3
+                          hasWallet={Boolean(walletWeb3)}
```

Comment:
I definitely don't remember exactly from where the `account` and other `use-wallet` related props are being drilled from, but we could definitely pass the `connected` prop corresponding to the wallet from `use-wallet`.